src/main/java/seedu/address/model/Goods.java

```diff
@@ -0,0 +1,18 @@
+package seedu.address.model;
+
+public class Goods {
+
+    private String name;
+
+    public Goods(String name) {
+        this.name = name;
+    }
```

Comment:
Maybe you can use "description" in place of "name" so that it is more flexible where you can include things like quantity, color, size, etc. 
I suggest abstracting Description to a new class and include constraints on the string itself such as limit of 255 characters can make it more OOP. You can refer to `Phone` class for its VALIDATION_REGEX for character limits on string and MESSAGE_CONSTRAINTS for error handling.

Reply:
Sure, I will try making it more OOP. Thanks for you suggestion!